service/src/ConstConfigIOGroup.hpp

```diff
@@ -88,6 +88,7 @@ namespace geopm
                 int domain;
                 std::function<double(const std::vector<double> &)> agg_function;
                 std::string description;
+                bool common_value_provided;
```

Comment:
`common_valud_provided` -> `is_common_value` or `is _common_value_provided`

I personally prefer is_common_value, but i'm okay with the longer name.  I try to enforce prefixing bool variable names with short verbs (usually a "to be verb") to aid with reading logic statements in code.  There are a couple of similar stack variables in the parser.